### ISSUE TITLE
"Occurrences" from Strong's not showing on iPad/macOS

### ISSUE THREAD

**Issue by benny-danny** [erykjj/jwpublib#44]
Hi Eryk.

I don't know if this is a bug or not but I've tried to view the occurrences of certain word in the Inline/Interlinear Bible and it is not shown on both my iPad and Macbook. But on my android phone it shows without any issue. See screenshots bellow. Maybe the issue is because it's a bigger screen?

Screenshots:
macOS/ipadOS (bigger screen)
<img width="1387" alt="SCR-20240629-kdwh" src="https://github.com/erykjj/jwpublib/assets/14347923/f46d4ae0-c4d4-4986-8e6f-bd042f44b0c6">
<img width="1387" alt="SCR-20240629-kdyf" src="https://github.com/erykjj/jwpublib/assets/14347923/98038878-4d21-4adb-943d-a8d684d62359">

Android phone (smaller screen)

![photo_2024-06-29 11 16 57](https://github.com/erykjj/jwpublib/assets/14347923/c3f47703-1149-42ee-9a46-83efda074c8e)
![photo_2024-06-29 11 17 03](https://github.com/erykjj/jwpublib/assets/14347923/bc9e67ce-3d0a-4e10-a525-0dd3b93f802e)

**Comment by erykjj** [comment]
That's a strange one. Has it just started behaving like this recently?

What version of JW Library do you have on your iPad? What version of Strong's?

I think what I would do is delete the Strong's publication from JW Library, restart the device, and install the Strong's jwpub again. Have you already tried this?

**Comment by benny-danny** [comment]
Yes, already tried that. It happens on both Vine's  and Strong's publications, on both iPadOS and macOS. 
Already uninstalled and installed again. Still the same issue. 
JWLibrary v. 15.0.1
Strong's v3.1
Vine's v3.1

**Comment by erykjj** [comment]
And is this recent? I mean, as of v15 of the JW Library?

Can you also check the Timeline of Bible History jwpub for me on your iPad: in any thematic/specific timeline, click the year. It should open the study pane with a "Year in context" view. Does that work?

**Comment by benny-danny** [comment]
> And is this recent? I mean, as of v15 of the JW Library?
> 
> Can you also check the Timeline of Bible History jwpub for me on your iPad: in any thematic/specific timeline, click the year. It should open the study pane with a "Year in context" view. Does that work?

To be honest, I've only came across your GitHub page a week ago, after installed v15, so, I cannot give feedback from previous versions.

About your question regarding TBH, it works as expected, I think.
![photo_2024-06-29 13 13 32](https://github.com/erykjj/jwpublib/assets/14347923/6bedff68-5070-4cdd-adff-12f341226c00)

**Comment by erykjj** [comment]
Thank you for confirming that TBH works.

You also mentioned that "it" happens also with Vine's? How so?

Let me confirm with another iPad user and think about your issue ;-)

**Comment by erykjj** [comment]
Just to confirm also: which version of KJS do you have installed?

**Comment by erykjj** [comment]
And, when you open Strong's directly (from Library > Downloaded) and go to a word in the main pane and click occurrences there?

**Comment by benny-danny** [comment]
> Thank you for confirming that TBH works.
> 
> You also mentioned that "it" happens also with Vine's? How so?
> 
> Let me confirm with another iPad user and think about your issue ;-)

Sorry, the Vine's mention has nothing to do with this error.. it was my mistake...



> Just to confirm also: which version of KJS do you have installed?

Both inline and interlinear. 
On my android phone I have both and they are working fine.



> And, when you open Strong's directly (from Library > Downloaded) and go to a word in the main pane and click occurrences there?

Still the same issue

![photo_2024-06-29 13 43 29](https://github.com/erykjj/jwpublib/assets/14347923/b8a81e89-980e-4250-9723-3878d3a7f121)

**Comment by erykjj** [comment]
In reality, the KJS Inline/Interlinear has nothing to do with it. The "Occurrences" view is part of Strong's. I will change the issue title to reflect that and await confirmation from another user. It _could_ be an issue with the recent update to JW Library.

**Comment by benny-danny** [comment]
> In reality, the KJS Inline/Interlinear has nothing to do with it. The "Occurrences" view is part of Strong's. I will change the issue title to reflect that and await confirmation from another user. It _could_ be an issue with the recent update to JW Library.

Yes, it could, but on android I'm also on JWlibrary v 15.0.1 and no issues so far.

**Comment by erykjj** [comment]
One more check: does the same happen with occurrences of Greek words?

**Comment by benny-danny** [comment]
> One more check: does the same happen with occurrences of Greek words?

Yes. Same issue. Also happens on my MacBook, so it’s no just on my iPad.

![IMG_0003](https://github.com/erykjj/jwpublib/assets/14347923/3d17d34e-bbf6-4a50-882f-f5e8e4c5eac1)
![IMG_0004](https://github.com/erykjj/jwpublib/assets/14347923/abcfbb8e-3c06-4748-80a2-ddc8c20f67df)

**Comment by erykjj** [comment]
I just checked on Windows (still not updated to v15) and noticed something that is also not showing correctly even on Android:
![2024-06-29_09-41-16](https://github.com/erykjj/jwpublib/assets/107011362/d0e4681d-1227-4c75-9ab4-c7deeaf9ee89)

So this could very likely be the result of the JW Library update :-/

**Comment by erykjj** [comment]
@benny-danny 

Please test this:
[Str_TEST.zip](https://github.com/user-attachments/files/16040792/Str_TEST.zip)

Change the extension to .jwpub

**Comment by erykjj** [comment]
OK, confirmed to be caused by the recent JW Library update. Stay tuned.

**Comment by benny-danny** [comment]
> OK, confirmed to be caused by the recent JW Library update. Stay tuned.

Thanks for the feedback! Glad you've found the issue.

**Comment by erykjj** [comment]
I've got a few things I can try. I've got the issue with Android not displaying the header correctly fixed. What you report may be due to something more extensive. I hope to have a fix for you to try tomorrow.

**Comment by erykjj** [comment]
@benny-danny 

Please test this when you get a chance:
[Str_beta.zip](https://github.com/user-attachments/files/16042700/Str_beta.zip)
Change suffix to .jwpub

**Comment by benny-danny** [comment]
> @benny-danny
> 
> Please test this when you get a chance: [Str_beta.zip](https://github.com/user-attachments/files/16042700/Str_beta.zip) Change suffix to .jwpub

Teste on MacBook and worked. Thanks.
<img width="1387" alt="SCR-20240630-iqzy" src="https://github.com/erykjj/jwpublib/assets/14347923/873f6a1b-01b5-4672-924a-1bfe39a639d8">

But yesterday (before I've tried this version) I've noticed another issue that is not present on android.
When I try to click on H3068, it does not open because it's "highlighted". The other H* open fine, but H3068 does not work. Issue is present in iPadOS and macOS, it previous version and also the beta version you've sent.
